### ISSUE TITLE
Would it be impossible to create a compatibility layer for React code?

### ISSUE THREAD

**Issue by kmcaloon** [BuilderIO/qwik#640]
### Is your feature request related to a problem?

One possible hurdle that might make developers slow to seriously consider Qwik is it's incompatibility with React, thus having to learn new hooks as well as not being able to use components/tools from the huge React ecosystem. Learning a slightly new syntax is easy, but not being able to leverage tools or libraries when creating decently robust UIs is going to be an issue.

### Describe the solution you'd like

Having a compatibility layer similar to how Preact accomplishes this via `preact/compat` would be amazing. I highly doubt that this would be logistically realistic but was wondering if there has been any talk about leveraging Babel to transform React syntax into Qwik.

### Describe alternatives you've considered

As far as I know, the only alternative would be to manually rewrite React components into Qwik. The drawbacks here are pretty obvious.

**Comment by nnelgxorz** [comment]
I think it's being worked on #493. The first task with the optimizer needing to process npm packages was just finished.

**Comment by kmcaloon** [comment]
Thanks @nnelgxorz. Trying to find more info in discord and the repo. I remember Misko mentioning something like this in his talk with Jason Lengstorf, but my understanding is that the limitation would be that it would actually be rendered as React correct? So sort of how Astro isolates and renders React components?

**Comment by mhevery** [comment]
Yes this is duplicate of https://github.com/BuilderIO/qwik/issues/493 Closing as duplicate.
